docs/StorageClass.puml

```diff
@@ -0,0 +1,27 @@
+@startuml
+class Storage {
+  - filePathString: String
+  - storageFile: File
+
+  + Storage(filePath: String)
+  + saveBoard(board: ChessBoard, playerColor: Color): void
+  + resetBoard(): void
+  + loadBoard(): ChessTile[][]
+  + loadPlayerColor(): Color
+
+  - createChessMasterFile(): void
+}
+
+class ChessBoard {
+  - SIZE: int
+}
+
+class ChessTile
+
+class Color
+
+Storage -[hidden]-> File
```

Comment:
Instead of doing a hidden connection to File, you could consider removing File from the parameters of Storage and unhide the hidden arrow